presto-main/src/main/java/io/prestosql/operator/LookupJoinPageBuilder.java

```diff
@@ -145,7 +145,7 @@ private void appendProbeIndex(JoinProbe probe)
     {
         int position = probe.getPosition();
         verify(position >= 0);
-        int previousPosition = probeIndexBuilder.isEmpty() ? -1 : probeIndexBuilder.get(probeIndexBuilder.size() - 1);
```

Comment:
Are there other collections like that with this issue?

Reply:
Based on profiling, I haven't found it yet. I just tried to find all `IntArrayList` usage here